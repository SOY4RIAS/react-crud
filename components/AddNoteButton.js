import React from 'react'
import Link from 'next/link'

const AddNoteButton = () => (
  <Link href="/add">
    <a
      className="button border-0 bg-green-600 text-white text-xl"
      data-testid="button"
      data-cy="add-note"
    >
      Add a note
    </a>
  </Link>
)

export default AddNoteButton
