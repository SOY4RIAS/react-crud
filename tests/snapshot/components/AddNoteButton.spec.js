import React from 'react'
import { render } from '@testing-library/react'
import '@testing-library/jest-dom/extend-expect'
import AddNoteButton from '~/components/AddNoteButton'

describe('Add Note Button', () => {
  it('should render the button', () => {
    const { container } = render(<AddNoteButton />)
    expect(container.firstChild).toMatchSnapshot()
  })
})
