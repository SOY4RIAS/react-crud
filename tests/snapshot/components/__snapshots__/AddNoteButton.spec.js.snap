// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Add Note Button should render the button 1`] = `
<a
  class="button border-0 bg-green-600 text-white text-xl"
  data-cy="add-note"
  data-testid="add-note"
  href="/add"
>
  Add a note
</a>
`;
